miio/integrations/vacuum/roborock/tests/test_vacuum.py

```diff
@@ -443,10 +444,63 @@ def test_cleaning_brush_cleaned_count(self):
         """Test getting cleaning brush cleaned count."""
         assert self.device.consumable_status().cleaning_brush_cleaned_count == 44
 
+    def test_mop_dryer_model_check(self):
```

Comment:
What do you think about simplifying this using pytest.mark.parametrize?
Something like this (untested):
```
@pytest.mark.parametrize("method_name,params", [("mop_dryer_settings", None), ("set_mop_drier_enabled", {"enabled": True}), ...])
def test_mop_drier_raises_unsupported(method_name, params):
    with pytest.raises(UnsupportedFeatureException):
        method = getattr(self.device, method_name)
        if params is not None:
            method(**params)
         else:
            method()
```

Reply:
AFAIK `@pytest.mark.parametrize` is not usable when using `TestCase` because the reference to `self` is missing.